nxttxt.py

```diff
@@ -0,0 +1,51 @@
+from pathlib import Path
+import exceptions
+
+
+def is_valid_path(path: str, should_be_dir: bool) -> bool:
+    if not path:
+        result = False
+    elif not Path(path).exists():
```

Comment:
if you wanted you could probably combine everything that  is result=False into one.